src/bionics_ui.cpp

```diff
@@ -36,6 +37,94 @@ enum bionic_menu_mode {
     EXAMINING,
     REASSIGNING
 };
+
+enum bionic_sort_mode {
+    POWER,
+    NAME,
+};
+bionic_sort_mode _bionic_sort_mode = bionic_sort_mode::POWER;
+
+auto sort_mode_str( bionic_sort_mode mode ) -> std::string
+{
+    std::string ret;
+    switch( mode ) {
+        case bionic_sort_mode::POWER:
+            ret = _( "Power Usage" );
+            break;
+        case bionic_sort_mode::NAME:
+            ret = _( "Name" );
+            break;
+    }
+    return ret;
+}
```

Comment:
There is standard enum-to-string infrastructure, but it's intended for serialization, not returning translated strings, so it doesn't really fit here.

Reply:
There is a nearly identical function in `advanced_inventory::get_sortname` so I think I can leave this as it is. Is that ok?